components/knative-function-controller/README.md

```diff
@@ -0,0 +1,102 @@
+# Knative Function Controller
+
+## Overview
+
+The Knative Function controller is a Kubernetes Controller that enables Kyma to manage Function resources.
+
+It defines and handles a Function Custom Resource Definition with the help of Knative Build and Knative Serving. Basically it is the serverless implementation based on Knative.
+
+## Prerequisites
+
+- Knative Build (v0.6.0)
+- Knative Serving (v0.6.1)
+- Istio (istio-1.0.7)
+
+## Installation
+
+### Local Deployment
+
+#### Manager running locally.
+
+Modify config/config.yaml to include your docker.io credentials (base64 encoded) and update the Docker registry value to your docker.io username.
+
+Apply the configuration
+
+```bash
+kubectl apply -f config/config.yaml
+```
+
+Install the CRD to a local Kubernetes cluster:
+
+```bash
+make install
+```
+
+Run the controller on your machine:
+
+```bash
+make run
+```
+
+#### Manager running inside k8s cluster
+
+This workflow needs to be used until kubernetes-sigs/kubebuilder#40 is fixed.
```

Comment:
The link to this issue doesn't work. Could you provide me with the valid link? I would include some information on this issue in our doc.
If the issue is fixed, would there be another approach to running the controller on the cluster?

Reply:
Done!